### ISSUE TITLE
grid_snap crashes Chrome and Edge

### ISSUE THREAD

**Issue by MSCAU** [IonDen/ion.rangeSlider#438]
Hi Den,

It looks as though **grid_snap: true** crashes the browser with 2.1.6. Seems to get caught in a loop. I have tested this with latest Chrome and Edge:

```
<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
<script>
  $(function() {
  	$("#slider_budget").ionRangeSlider({
	    type: "double",
	    // grid_snap: true,  // This line crashes the browser
	    grid: true,
	    min: 0,
	    max: 80000,
	    from: 2000,
	    to: 30000,
	    prefix: "$ ",
	    hide_min_max: true,
            prettify_enabled: true,
    	    prettify_separator: ","
   	});
  });
</script>
```

I don't know why it works on your demo page...

**Comment by IonDen** [comment]
Hi, thanks for finding this. This is happens because of too much calculations. Check this demo:
http://jsfiddle.net/IonDen/7hhasqyo/ if will see that it is looking ugly already.

So, the best way to handle the problem is to add STEP to config, like this:
http://jsfiddle.net/IonDen/g73upq85/

To avoid this problem in the future i will add some check to the code.

**Comment by MSCAU** [comment]
OK thanks. That will do for now.

**Comment by IonDen** [comment]
Fixed in 2.1.7
